static/app/views/starfish/modules/databaseModule/databaseTableView.tsx

```diff
@@ -70,34 +80,60 @@ export default function APIModuleView({
   location,
   data,
   onSelect,
+  onSortChange,
   selectedRow,
   isDataLoading,
   columns,
 }: Props) {
-  function renderHeadCell(column: GridColumnHeader): React.ReactNode {
-    return <span>{column.name}</span>;
+  const [sort, setSort] = useState<{
+    direction: 'desc' | 'asc' | undefined;
+    sortHeader: TableColumnHeader | undefined;
+  }>({direction: undefined, sortHeader: undefined});
+
+  function onSortClick(col: TableColumnHeader) {
+    let direction: 'desc' | 'asc' | undefined = undefined;
+    if (sort.direction === 'desc') {
+      direction = 'asc';
+    } else if (!sort.direction) {
+      direction = 'desc';
+    }
+    if (onSortChange) {
+      setSort({direction, sortHeader: col});
+      onSortChange({direction, sortHeader: col});
+    }
+  }
+
+  function renderHeadCell(col: TableColumnHeader): React.ReactNode {
+    const sortableKeys: Keys[] = ['p75', 'epm', 'total_time', 'domain', 'transactions'];
+    if (sortableKeys.includes(col.key)) {
+      const isBeingSorted = col.key === sort.sortHeader?.key;
+      const direction = isBeingSorted ? sort.direction : undefined;
+      return (
+        <SortableHeader
+          onClick={() => onSortClick(col)}
+          direction={direction}
+          title={col.name}
+        />
+      );
+    }
+    return <span>{col.name}</span>;
   }
 
   function renderBodyCell(
-    column: GridColumnHeader,
+    column: TableColumnHeader,
     row: DataRow,
     rowIndex: number
   ): React.ReactNode {
+    const {key} = column;
+
     const isSelectedRow = selectedRow?.group_id === row.group_id;
     const rowStyle: CSSProperties | undefined = isSelectedRow
       ? {fontWeight: 'bold'}
       : undefined;
 
-    if (column.key === 'columns') {
-      const value = row.data_values[row.data_keys.indexOf('columns')];
-      return value ? <ArrayValue value={value?.split(',')} /> : <span />;
-    }
-    if (column.key === 'order') {
-      const value = row.data_values[row.data_keys.indexOf('order')];
-      return value ? <ArrayValue value={value?.split(',')} /> : <span />;
-    }
```

Comment:
@wmak fyi, these were unused, so I removed them, but lmk if we should keep it for the future.